.vscode/settings.json

```diff
@@ -2,6 +2,9 @@
   "editor.defaultFormatter": "esbenp.prettier-vscode",
   "editor.formatOnSave": true,
   "editor.codeActionsOnSave": {
-    "source.organizeImports": true
+    "source.organizeImports": "explicit"
+  },
+  "yaml.schemas": {
+    "https://json.schemastore.org/github-workflow.json": "file:///home/michal/dev/ohn/sleepy.bike/sleepy.bike/.github/workflows/deploy.yml"
```

Comment:
looks like a path on your local desktop got in here accidentally.

Reply:
Thanks for pointing that out! Fixed in ecf32ecfd9f583916be39f8